spec/index.bs

```diff
@@ -6222,6 +6225,7 @@ dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
     required USVString code;
     object sourceMap;
     record<USVString, GPUShaderModuleCompilationHint> hints;
+    boolean fragmentShaderDerivativesCheckedUniformity = true;
```

Comment:
u-nit: I think in general it is better to make default values false if we can because `undefined` may look like it would coerce to `false`, so `uncheckedFragmentShaderDerivativeUniformity`?

Reply:
Right.

Normally I try to define things in the positive sense because I find them easier to understand.  I find things like "NDEBUG" confusing.

But I can see that JS practice could supercede that rule of thumb.